spec/specification/json_spec.rb

```diff
@@ -16,6 +16,7 @@ module Pod
             'tvos' => nil,
             'watchos' => nil,
           },
+          'test_specification' => false,
```

Comment:
Yeah I don't think we should write this to the hash unless it's true, or even better, for it to always be computed from `test_type`, which would maybe give us this for free too?